packages/core/src/auth/ee/fga-check.ts

```
@@ -5,6 +5,7 @@
  */
 
 import { captureEEEvent, getEETelemetryFallbackDistinctId } from '../../telemetry/posthog';
+import type { MCPServerFGAConfig } from '../../mcp/types';
```

ESLint

[error] `../../mcp/types` type import should occur before import of `../../telemetry/posthog`

(import/order)